projects/angular/src/emphasis/alert/alerts-pager.ts

```diff
@@ -45,6 +45,12 @@ export class ClrAlertsPager implements OnInit, OnDestroy {
     return this.multiAlertService.current;
   }
 
+  get alertsCurrentPageAriaLabel() {
+    return `${this.commonStrings.keys.alertCurrentAlertAriaLabel}, ${this.multiAlertService.current + 1} of ${
+      this.multiAlertService.count
+    }`;
```

Comment:
The `alertCurrentAlertAriaLabel` would need to be a template. The "X of Y" needs to be translated properly.

Reply:
The `X/Y` text is visible on the page, so I am not sure if it's needed in the `aria-label` anyway. cc: @AmyLiNow

Reply:
I added it to be as additional to the buttons. So once one of the two arrow is focused then the screen reader would say "Next Alert, button" then "Current alert, 1 of 4". Otherwise if this element is used without the additional label it would say "1 slash 4" which would not be exactly clear for someone who is using it.

Reply:
Okay, we should make sure to get a review from Amy on this.

Reply:
Technically there is no WCAG violation but you can argue that it can be interpreted as math because of the slash, so Daniel is right, it is confusing and unclear for those who use screen readers. 

I see this currently `<div id="alerts-pager" class="alerts-pager-text" aria-label="Current alert, 1 of 4"> 1 / 4 </div>`
`aria-label ` only works(some screen readers are starting to be more forgiving and still read aria-labels with not dependent role) when the HTML element/tag has an identity that can be named (<button>, <a>, <header>, <nav>, etc) or an ARIA `role` ( button, link, navigation, region etc). 
In this is just a `div`. 

Recommendations (#2 is ideal):
1. At a minimum, I recommend changing "1/4" to "1 of 4" as it could be read as math division. No aria would be needed here.
2. If possible I recommend a combination of visually hidden text "Current alert, " and visually text "1 of 4". so it would look like this: `<span class="screen-reader-only">Current Alert, </span><span> 1 of 4</span>`. This also will not require aria.